src/control/lib/spdk/src/nvme_control.c

```diff
@@ -81,52 +87,20 @@ nvme_discover(void)
 	return _discover(&spdk_nvme_probe, true, &get_health_logs);
 }
 
-static void
-read_complete(void *arg, const struct spdk_nvme_cpl *completion)
-{
-	struct lba0_data *data = arg;
-
-	spdk_free(data->buf);
-
-	if (spdk_nvme_cpl_is_success(completion)) {
-		data->write_result = LBA0_WRITE_SUCCESS;
-		return;
-	}
-
-	spdk_nvme_qpair_print_completion(data->ns_entry->qpair,
-					 (struct spdk_nvme_cpl *)completion);
-	fprintf(stderr, "I/O error status: %s\n",
-		spdk_nvme_cpl_get_status_string(&completion->status));
-	fprintf(stderr, "Read I/O failed, aborting run\n");
-	data->write_result = LBA0_WRITE_FAIL;
-}
-
+/** callback for write command completion when wiping out a ns */
 static void
 write_complete(void *arg, const struct spdk_nvme_cpl *completion)
 {
-	struct lba0_data	*data = arg;
-	struct ns_entry		*ns_entry = data->ns_entry;
-	int			 rc;
+	struct lba0_data *data = arg;
 
 	if (spdk_nvme_cpl_is_success(completion)) {
-		rc = spdk_nvme_ns_cmd_read(ns_entry->ns, ns_entry->qpair,
-					   data->buf, 0, 1, read_complete,
-					   (void *)data, 0);
-		if (rc != 0) {
-			fprintf(stderr, "starting read I/O failed (%d)\n", rc);
-			data->write_result = LBA0_WRITE_FAIL;
-		}
-		return;
+		data->result = LBA0_WRITE_SUCCESS;
+	} else {
```

Comment:
nit, Rather than providing else clause and indenting block, just return after setting success and dedent the rest